.github/workflows/release.yaml

```diff
@@ -132,6 +134,43 @@ jobs:
             chainloop attestation add --name $material_name --value $entry --kind ARTIFACT --attestation-id ${{ env.ATTESTATION_ID }}
           done
 
+      - name: Calculate checksum for SLSA attestation
```

Comment:
I'd add a link explaining why we need to do this. Pointing to the issue that mentioned that only one container image is possible at the moment